modules/calc/src/main/java/com/opengamma/strata/calc/runner/CalculationFunction.java

```diff
@@ -103,6 +103,8 @@
    * Determines the market data required by this function to perform its calculations.
    * <p>
    * Any market data needed by the {@code calculate} method should be specified.
+   * <p>
+   * The set of measures may include measures that are not supported by this function.
```

Comment:
Do you not think it's worth filtering the measures passed to the `requirements` method too?